bootcamp/tutorials/quickstart/use_ColPALI_with_milvus.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>Add an illustrative image to show what colpali is, like what's used in the paper.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>